7-kubernetes-container-orchestration/7.7-k8sthw.md

```diff
@@ -0,0 +1,79 @@
+# K8s the Hard Way
+In this section, we will be taking a deep dive into the building blocks of Kubernetes, following along with Ivan Fioravanti's fork of [Kubernetes the Hard Way](https://github.com/ivanfioravanti/kubernetes-the-hard-way-on-azure). We will be creating the Terraform and Ansible code along the way that will manually stand everything up within Azure.
+
+This way of creating a cluster is by no means production-ready and is designed solely for learning the low level, administrative perspective. This implies going the extra mile to make sure you comprehend each step involved in setting up a Kubernetes cluster. (Go into more detail about understanding the k8s control plane and all the different components)
+
+## A Quick High Level of the Kubernetes Control Plane
```

Comment:
Looks good, maybe some links to the Kubernetes documentation for the scheduler, api-server etc.